src/components/BnFileInput/BnFileInput.styles.cjs

```diff
@@ -10,10 +10,7 @@ module.exports = {
       '@apply p-0 border-0 h-auto bg-banano-bg text-banano-text-foreground': {},
     },
     '&--error': {
-      '@apply border border-red-300 text-red-700 ring-red-300 outline-red-300 !important': {},
-      '.bn-file-input__placeholder': {
-        '@apply text-red-500': {},
-      },
```

Comment:
está bien que se borre esto?

Reply:
I removed it following the library figma, in the section Labels, the inputs' texts don't change to red when there is an error state. The figma doesn't have an example with a placeholder, but I assumed it would be the same as with texts